sample/src/main/res/layout/fragment_pie.xml

```diff
@@ -6,17 +6,13 @@
     android:layout_height="match_parent"
     tools:context=".ui.pie.PieFragment">
 
-    <TextView
-        android:id="@+id/text_notifications"
+    <androidx.compose.ui.platform.ComposeView
+        android:id="@+id/chartPie"
```

Comment:
for xml pls use snake case and for classes/activiites/fragment pls use camelCase

* to determine that this is from xml layout